Aptoide/aptoide-client-v8
# [APP-2337] Implement search

**What does this PR do?**

This implements
- [ ] Bottom Navigation Item
- [ ] Recent Searches
- [ ] Popular Searches
- [ ] Search Suggestions (With v10 autocomplete, not buzz yet)
- [ ] Search results (higlighted result aswell)

As said in the ticket, as well as appview navigation. It adds some assets as well (fonts, colors, icons)... 
For analytics only a print to the console is being done.

As also said in the ticket, the install wasn't implemented.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AppColors.kt
- [ ] AppGamesAutoCompleteSuggestionsRepository.kt
- [ ] AppGamesSearchStoreManager.kt
- [ ] AppIcons.kt
- [ ] AppItems.kt
- [ ] AppTypography.kt
- [ ] AutoCompleteSuggestion.kt
- [ ] BottomBarMenus.kt
- [ ] build.gradle.kts
- [ ] Color.kt
- [ ] EmptyView.kt
- [ ] Gamepad.kt
- [ ] HistoryOutlined.kt
- [ ] InstallViewShort.kt
- [ ] ProgressText.kt
- [ ] RepositoryModule.kt
- [ ] roboto_condensed_bold.ttf
- [ ] roboto_condensed_bolditalic.ttf
- [ ] roboto_condensed_italic.ttf
- [ ] roboto_condensed_light.ttf
- [ ] roboto_condensed_lightitalic.ttf
- [ ] roboto_condensed_medium.ttf
- [ ] roboto_condensed_mediumitalic.ttf
- [ ] roboto_condensed_regular.ttf
- [ ] SearchType.kt
- [ ] SearchView.kt
- [ ] strings.xml

**How should this be manually tested?**
 Test the whole search flow and its different types (related, manual, popular and suggestions), see if everything's working as intended and nothing's broken
  Flow on how to test this or QA Tickets related to this use-case: [APP-2337](https://aptoide.atlassian.net/browse/APP-2337)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2337](https://aptoide.atlassian.net/browse/APP-2337)


**Code Review Checklist**
